.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
That sounds good, but IMO we should accept fixes via [PR](https://github.com/blog/1506-closing-issues-via-pull-requests) as well.  Some fixes may span multiple commits, and if someone forgets, it is troublesome to edit a commit message - much easier to update a PR.  The contributor will also see the reminder in the PR template, which they'd usually see after they've committed changes.  Personally, I usually forget and need to update the PR.  ;)